src/main/java/com/amalitech/gpuconfigurator/model/attributes/Attribute.java

```diff
@@ -0,0 +1,62 @@
+package com.amalitech.gpuconfigurator.model.attributes;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.UUID;
+
+@Data
```

Comment:
Don't use @Data annotation in classes marked with @Entity
Use @Getter, @Setter and define your custom Equals and HashCode in the class